src/tools/sfdk/dispatch.cpp

```diff
@@ -55,6 +57,11 @@ const char TYPE_KEY[] = "type";
 const char VERSION_KEY[] = "version";
 const char WORKER_KEY[] = "worker";
 
+const char OS_VARIANT_MACRO[] = "%{OsVariant}";
+const char OS_VARIANT_NOSPACE_MACRO[] = "%{OsVariant:NoSpace}";
+const char OS_VARIANT_UNDERLINE_MACRO[] = "%{OsVariant:Underline}";
+const char IDE_VARIANT_MACRO[] = "%{IdeVariant}";
+const char SDK_VARIANT_MACRO[] = "%{SdkVariant}";
```

Comment:
There is a nice blank line at the beginning of the block - I would add one here as well.